src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { PreparedPosts } from '../types/User';
+import { CommentList } from '../Comment/CommentList';
+import { UsersInfo } from '../UsersInfo/UsersInfo';
+import './PostInfo.scss';
+
+type Props = {
+  post: PreparedPosts;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__post">
+        <strong className="PostInfo__name" data-cy="post-title">
+          {post?.title.toUpperCase()}
+        </strong>
+
+        <p className="PostInfo__item" data-cy="post-body">
+          {post?.body}
+        </p>
```

Comment:
Will you render empty p if the post doesn't have a body?

Reply:
too